dts/bindings/gpio/microchip,mcp230xx.yaml

```diff
@@ -5,9 +5,9 @@
 #
 
 description: |
-    This is a representation of the Microchip MCP23008 I2C Gpio Expander.
+    This is a representation of the Microchip MCP230xx I2C Gpio Expander.
 
-compatible: "microchip,mcp23008"
+compatible: "microchip,mcp230xx"
 
```

Comment:
This will break out of tree DTS files; can you please add something to the release notes similar to what I did in 1f3ca8212651d8b0f0b5b57a8891142526b050a7? Note "similar" because the change is for different reasons here, so the wording will need to change slightly although the advice will be the same.

Reply:
Even though that driver/compatible was never in a proper Zephyr release? Does Zephyr promise compat for things found only in `main`?

Reply:
Oh, I didn't realize that. Please disregard then. Thanks.